docs/admin/system-information.rst

```diff
@@ -2266,6 +2266,100 @@ Alternatively the statistics can also be updated using the :ref:`ANALYZE
+|                  | ``true``                                 |             |
++------------------+------------------------------------------+-------------+
+
+.. _pg_publication_tables:
+
+pg_publication_tables
+=====================
+
+The ``pg_publication_tables`` table in the ``pg_catalog`` system schema
+contains tables replicated by the publication.
+
++----------------+--------------------------------------+-------------+
+| Column Name    | Description                          | Return Type |
++================+===+==================================+=============+
+| ``pubname``    | Name of the publication.             | ``TEXT``    |
++----------------+--------------------------------------+-------------+
+| ``schemaname`` | Name of the schema containing table. | ``TEXT``    |
++----------------+---+---------------------+------------+-------------+
+| ``tablename``  | Name of the table.                   | ``TEXT``    |
++----------------+--------------------------------------+-------------+
+
+
+.. _pg_subscription:
+
+pg_subscription
+===============
+
+The ``pg_subscription`` table in the ``pg_catalog`` system schema contains all
+subscriptions created in the cluster.
+
++---------------------+------------------------------------------+-------------+
+| Column Name         | Description                              | Return Type |
++=====================+==========================================+=============+
+| ``oid``             | Row identifier.                          | ``INTEGER`` |
++---------------------+------------------------------------------+-------------+
+| ``subdbid``         | noop value, always ``0``.                | ``INTEGER`` |
++---------------------+------------------------------------------+-------------+
+| ``subname``         | Name of the subscription.                | ``TEXT``    |
++---------------------+------------------------------------------+-------------+
+| ``subowner``        | Oid of the owner of the subscription.    | ``INTEGER`` |
++---------------------+------------------------------------------+-------------+
+| ``subenabled``      | Whether subscription is enabled, always  | ``BOOLEAN`` |
+|                     | ``true``.                                |             |
++---------------------+------------------------------------------+-------------+
+| ``subbinary``       | Noop value, always ``true``.             | ``BOOLEAN`` |
+|                     |                                          |             |
++---------------------+------------------------------------------+-------------+
+| ``substream``       | Noop value, always ``true``.             | ``BOOLEAN`` |
+|                     |                                          |             |
++---------------------+------------------------------------------+-------------+
+| ``subconninfo``     | Connection string to the publishing      | ``TEXT``    |
```

Comment:
we currently use `r.subscription.connectionInfo().hosts()` - probably need more info, ie
make it look like `crate://example.com?user=my_user&password=1234&sslmode=disable&seeds=123.123.123.123,123.123.123.124`?

Reply:
Not sure - we'd risk leaking credentials? We'd have to reat it similar to the password within the users table.

Reply:
I think the other information would be good to expose.